kubectl-plugin/kubectl-waiops

```diff
@@ -474,9 +527,7 @@ then
         echo ""
         exit 0
     else       
-        printf "${blue}${bold}Hint: log into your cluster with CLUSTER_ADMIN credentials and try again.${normal}
-
-"
+        printf "${blue}${bold}Hint: log into your cluster with CLUSTER_ADMIN credentials and try again.${normal}"
```

Comment:
I think we need to add those returns back in. Here's what I see when I run all three commands while not in cluster-admin mode (This line for `status-all` is the one with the formatting issue):

<img width="2257" alt="Screen Shot 2022-07-05 at 4 34 27 PM" src="https://user-images.githubusercontent.com/91081503/177433851-f2d9f0e8-1cc6-44f5-9a97-8644e6ddd66e.png">

Reply:
@kvijai82 wondering if we added these spaces back in.